docs/fundamentals/code-analysis/quality-rules/ca2241.md

```diff
@@ -46,10 +46,15 @@ Do not suppress a warning from this rule.
 
 Use the following options to configure additional methods to run this rule on.
 
-- [Additional string formatting methods](#additional-string-formatting-methods)
-- [Determine additional string formatting methods automatically](#determine-additional-string-formatting-methods-automatically)
-
-You can configure these options for just this rule, for all rules, or for all rules in this category (Usage). For more information, see [Code quality rule configuration options](../code-quality-rule-options.md).
+- [CA2241: Provide correct arguments to formatting methods](#ca2241-provide-correct-arguments-to-formatting-methods)
+  - [Cause](#cause)
+  - [Rule description](#rule-description)
```

Comment:
Something weird happened here. Do you have a tool to add the indexes automatically?

Reply:
I don't or at least not aware of it, I did the change in VS Code.

Reply:
I confirm it automatically happens on save in VS Code. It's not doing that for the other files...

Reply:
That's very strange. What extensions do you have installed, and what are the steps to get this to happen?